resources/checkstyle.xml

```diff
@@ -48,7 +48,7 @@
         </module>
         <module name="NoWhitespaceBefore"/>
         <module name="RedundantImport"/>
-        <module name="RedundantModifier"/>
+       <!-- <module name="RedundantModifier"/> -->
```

Comment:
The PR LGTM, but why are you removing this check?

Reply:
We have some static imports (search for "import static") in the code. I am doubting it is worth the effort to fix it. Wdyt ?

Reply:
I'm wondering why this check was not failing before. Anyway, I agree there is little value in this one.

Reply:
This check should find redundant modifiers like `public` in interfaces, not static imports.

Reply:
Right. I ran a local build and only in `core` there were more than 40 violations. I fixed them, then found the same in another project... So I agree the value this check provides is close to zero and I think removing it is OK.

Reply:
I run the report again, and yeah sorry you are right: The violations where way down the file not on top where the "import static" was used.